### DISCUSSION TITLE
Williams Alligator and Gator Oscillator

### DISCUSSION THREAD

**Discussion by tanjera** [DaveSkender/Stock.Indicators#385] [Indicators]
Community discussion about [Williams Alligtor](https://dotnet.stockindicators.dev/indicators/Alligator/#content) and [Gator Oscillator](https://dotnet.stockindicators.dev/indicators/Gator/#content)

**Comment by jgwill** [comment]
Any hint how to use it?  (I am familiar with the Alligator, the AO but not that one from Williams.  (I did his Profitunity Home Study Program).  Pretty interesting)
regards

**Comment by sshquack** [comment]
@DaveSkender I'm trying to add the Gator Oscillator to https://charts.stockindicators.dev/ but it doesn't show up as an option. Williams Alligator works but Gator Oscillator is missing. Could this also be related to https://github.com/DaveSkender/Stock.Indicators/discussions/477#discussioncomment-4492623 ?

**Reply by DaveSkender** [reply]
It's not integrated yet, see https://github.com/DaveSkender/Stock.Charts/issues/212
